react-api-client/src/camera/useCameraSettings.ts

```diff
@@ -0,0 +1,63 @@
+import { useMutation, useQueryClient } from 'react-query'
+
+import { createCameraSettings } from '@opentrons/api-client'
+
+import { useHost } from '../api'
+
+import type { AxiosError } from 'axios'
+import type {
+  UseMutateFunction,
+  UseMutationOptions,
+  UseMutationResult,
+} from 'react-query'
+import type {
+  CameraResponse,
+  CameraSettings,
+  ErrorResponse,
+} from '@opentrons/api-client'
+import type { CameraId } from '@opentrons/shared-data'
+
+export type UseUpdateCameraMutationResult = UseMutationResult<
+  CameraResponse,
+  AxiosError<ErrorResponse>,
+  CameraSettings
+> & {
+  updateCameraSettings: UseMutateFunction<
+    CameraResponse,
+    AxiosError<ErrorResponse>,
+    CameraSettings
+  >
+}
+
+export function useCameraSettings(
+  options: UseMutationOptions<
+    CameraResponse,
+    AxiosError<ErrorResponse>,
+    CameraSettings
+  > = {},
+  cameraId: CameraId
+): UseUpdateCameraMutationResult {
+  const host = useHost()
+  const queryClient = useQueryClient()
+
+  const mutation = useMutation<
+    CameraResponse,
+    AxiosError<ErrorResponse>,
+    CameraSettings
+  >(
+    [host, 'camera/settings'],
```

Comment:
```suggestion
    [host, 'camera', 'settings', cameraId],
```
React Query is dependent on query keys following a certain convention. If you aren't familiar with how they work, I'd read about them [here](https://tanstack.com/query/v5/docs/framework/react/guides/query-keys).